chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/SvapController.java

```diff
@@ -0,0 +1,75 @@
+package gov.healthit.chpl.web.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import gov.healthit.chpl.domain.CertificationCriterion;
+import gov.healthit.chpl.exception.EntityRetrievalException;
+import gov.healthit.chpl.exception.ValidationException;
+import gov.healthit.chpl.logging.Loggable;
+import gov.healthit.chpl.svap.domain.Svap;
+import gov.healthit.chpl.svap.manager.SvapManager;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Api(value = "svaps")
+@RestController
+@RequestMapping("/svaps")
+@Loggable
+public class SvapController {
+
+    private SvapManager svapManager;
+
+    @Autowired
+    public SvapController(SvapManager svapManager) {
+        this.svapManager = svapManager;
+    }
+
+    @ApiOperation(value = "Update an SVAP.",
+            notes = "Provides functionality to update an SVAP and the Criteria associated with it."
+                    + "Security Restrictions: To update: ROLE_ADMIN or ROLE_ONC.")
+    @RequestMapping(value = "", method = RequestMethod.PUT, consumes = MediaType.APPLICATION_JSON_VALUE,
+        produces = "application/json; charset=utf-8")
+    public @ResponseBody Svap updateSvap(@RequestBody(required = true) Svap svap) throws EntityRetrievalException, ValidationException {
+        return svapManager.update(svap);
+    }
+
+    @ApiOperation(value = "Create an SVAP.",
+            notes = "Provides functionality to add a new SVAP and the Criteria associated with it."
+                    + "Security Restrictions: To update: ROLE_ADMIN or ROLE_ONC.")
```

Comment:
I think there is maybe a copy/paste error in the notes although I could just be reading it incorrectly.  The first three methods here all say "To update:" but only the first method is doing an update.